Sail/src/API/DX12/shader/DX12StructuredBuffer.cpp

```diff
@@ -40,15 +69,15 @@ namespace ShaderComponent {
 	DX12StructuredBuffer::~DX12StructuredBuffer() {
 	}
 
-	void DX12StructuredBuffer::updateData(const void* newData, unsigned int numElements, int meshIndex) {
-		assert(numElements < MAX_ELEMENTS && "Too many elements! Increase MAX_ELEMENTS in DX12StrucutedBuffer.h");
+	void DX12StructuredBuffer::updateData(const void* newData, unsigned int numElements, int meshIndex, unsigned int offset) {
+		//assert(numElements < MAX_ELEMENTS && "Too many elements! Increase MAX_ELEMENTS in DX12StrucutedBuffer.h");
```

Comment:
Do you need to save this code?

Reply:
Yes, might be needed in future. Will add comment about it